Warn about potential infinite loop in on_construct (#12967)
minetest/minetest: doc/lua_api.txt

        -- Node constructor; called after adding node.
        -- Can set up metadata and stuff like that.
        -- Not called for bulk node placement (i.e. schematics and VoxelManip).
        -- Note: Within an on_construct callback, minetest.set_node can cause an

>         -- infinite loop if it invokes the same callback. It may be necessary to
>         -- use minetest.swap_node instead.

Reviewer: ```suggestion
        -- infinite loop if it invokes the same callback.
        --  Consider using minetest.swap_node instead.
```
